### PR TITLE
Fix other sickbeard forks errorring.

### PR DESCRIPTION
**Pull request by p0psicles**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)
- [x] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

**Test Configuration**:

# Checklist:
- [x] I have based this change on the nightly branch
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by p0psicles** [comment]
@clinton-hall it's ready imo. And I tested with medusa's three fork variations.
Should we update a wiki with instructions on how to use apiv2? And it's benifits?

File: core/auto_process/managers/sickbeard.py [hunk 1]
```diff
@@ -303,9 +300,13 @@ def detect_fork(self):
...
     def _init_fork(self):
-        from .pymedusa import PyMedusa
+        # These need to be imported here, to prevent a circular import.
+        from .pymedusa import PyMedusa, PyMedusaApiV1, PyMedusaApiV2
+
         mapped_forks = {
-            'Medusa': PyMedusa
+            'Medusa': PyMedusa,
+            'Medusa-api': PyMedusaApiV1,
+            'Medusa-apiv2': PyMedusaApiV2
```
Comment (p0psicles) [inline core/auto_process/managers/sickbeard.py:309]: you could add a line here for the default sickbeard fork. And use the class `SickBeard` for the same file. But i'm not sure for which "forks" you would want to use that. So I haven't added it

Comment (clinton-hall) [reply core/auto_process/managers/sickbeard.py:309]: Thanks, I'll take a look and see what forks to add here.

As to wiki, absolutely. Probably need to update https://github.com/clinton-hall/nzbToMedia/wiki/sickbeard-forks and create a new Medusa Page

**Comment by XxUnkn0wnxX** [comment]
no issues with PP using apiv2 -  8039bd

**Comment by p0psicles** [comment]
Note to myself.
- [ ] update wiki

**Pull request merged**